docs/sources/enterprise/reporting.md

```diff
@@ -95,7 +95,7 @@ If the time zone is set differently between your Grafana server and its remote i
 
 ### CSV export
 
-> **Note:** Only available in Grafana Enterprise v8.0+, with the [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) v3.0+.
+> **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) 8+ with the [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) v3.0+, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
```

Comment:
This line includes "8+" and "v3.0+" - not sure if our official format prepends a "v" or always includes the minor version to format software versions, but this is just a note to make them consistent.